### PR TITLE
[WC-730]: add docs for new image widget

### PR DESCRIPTION
**Pull request by Keraito**
- This is gonna be a new image widget
- Will be released in 9.6.0
- Because it's a new widget, the marketplace link is pending

### CODE REVIEW

File: content/appstore/widgets/image.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+* **Width unit** and **Width** – the width of the widget in relation to the rest of the elements on the page
+	* The available **Width unit** options are **Auto**, **Percentage** and **Pixels**
+	* The **Width** can be set as an appropriate CSS value when **Width unit** is not set to **Auto**
+	* These two properties need to be used together to work
+* **Height unit** and **Height** – the height of the widget in relation to the rest of the elements on the page
+	* The available **Height unit** options **Auto** and **Pixels**. **Auto** will keep the aspect ratio of the image
+	* The **Height** can be set as an appropriate CSS value when **Height unit** is not set to **Auto**
+	* These two properties need to be used together to work
+* **Show** (default: **Full image**) - Determines how the image will be shown. When set to **Thumbnail**, the widget will try to retrieve the thumbnail version of the image if present.
+* **Responsive** (default: **Yes**) - When set to **Yes**, the image will never get larger than its original size. It can become smaller.
```
Comment (IIsaku) [inline content/appstore/widgets/image.md:65]: Maybe explain here also what responsive means: growing with the parent container...

**Review by IIsaku — CHANGES_REQUESTED** [review]
Nice documentation. I added two points to have a look at.

File: content/appstore/widgets/image.md (hunk 1 above)
Comment (Keraito) [reply content/appstore/widgets/image.md]: Fixed

**Comment by Adam-Dupaski** [comment]
@Luyao-Zhang-1 assigning this to @ConnorLand instead, as he worked on the original PR.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**